astropy/utils/data.py

```diff
@@ -1370,3 +1370,27 @@ def _release_download_cache_lock():
         msg = 'Error releasing lock. "{0}" either does not exist or is not ' +\
               'a directory.'
         raise RuntimeError(msg.format(lockdir))
+
+
+def _urls_on_shelf():
+    """
+    Get the list of URLs in the shelf. Useful for looking up what files are
+    cached when you don't have internet access.
+
+
+    Returns
+    -------
+    in_cache : bool
```

Comment:
This looks like a copy-and-paste typo?  Shouldn't it be the list of URLs that are cached?